scanners/zap-extended/scanner/zap_configuration.py

```diff
@@ -0,0 +1,67 @@
+#
+# This file contains some ZAP hooks.
+#
+# See https://www.zaproxy.org/docs/docker/scan-hooks/ for more information.
+#
+
+import collections
+import logging
+import glob
+import hiyapyco
+
+class ZapConfiguration():
+    """This class represent a ZAP specific configuration based on a given YAML file"""
+    
+    def __init__(self, scantype_config_dir: str, scan_config_dir: str):
+        """Initial constructor used for this class"""
+        
+        self.__scantype_config_dir = scantype_config_dir
```

Comment:
Is there a specific reason to use double underscore here instead of a single underscore for the private variables/methods?

Reply:
Yes, the variable should be explicitly private  (double underscore) instead of private by convention (single underscore).